Makefile.build

```diff
@@ -379,32 +378,40 @@ coqbinaries: $(COQMKTOP) $(COQTOPEXE) $(CHICKEN) $(CSDPCERT) $(FAKEIDE)
 
```

Comment:
It seems that a reference to the `COQMKTOP` variable has been forgotten, on line 377.

Reply:
Thanks @vbgl !